cabal-install/main/Main.hs

```diff
@@ -302,7 +305,11 @@ mainWorker args = do
       , hiddenCmd  win32SelfUpgradeCommand win32SelfUpgradeAction
       , hiddenCmd  actAsSetupCommand actAsSetupAction
       , hiddenCmd  manpageCommand (manpageAction commandSpecs)
-
+      -- ghc-mod supporting commands
+      , hiddenCmd CmdShowBuildInfo.showBuildInfoCommand
```

Comment:
Needs reviewing, I dont know what kind of commands `new-show-build-info` is supposed to be.

Reply:
I think having it hidden for now is ok. AFAIK that just affects whether the command shows up in the command list in `--help` output. So if we don't want users to discover this yet we can keep it hidden until we're done fleshing it out.